### PR TITLE
Merge install-best-practices into yum repos doc (SOFTWARE-3078)

### PR DESCRIPTION
**Pull request by matyasselmeci**
Move the content from the install best practices doc into the yum repos doc and delete the install best practices doc. The only best practice that's missing was the advice to turn off yum autoupdate, which should be placed in some more prominent place anyway.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Jeez, give me a doc to review and I git rm it :)
I can do some squashing after this PR has been reviewed.

File: docs/common/yum.md [hunk 1]
```diff
@@ -5,122 +5,134 @@ About This Document
...
--   **release**: This repository contains software that we are willing to support and can be used by the general community.
--   **contrib**: RPMs contributed from outside the OSG.
--   **testing**: This repository contains software ready for testing. If you install packages from here, they may be buggy, but we will provide limited assistance in providing a migration path to a fixed version.
--   **development**: This repository is the bleeding edge. Installing from this repository may cause the host to stop functioning, and we will not assist in undoing any damage.
+-   **release**: RPMs considered production-ready.
+-   **testing**: RPMs not yet ready for release; expect bugs.
+-   **development**: RPMs that are bleeding-edge;
+      do not use without instruction from OSG Software and Release team members.
+-   **contrib**: RPMs contributed from outside the OSG S&R team;
+      no official OSG support.
```
Comment (brianhlin) [inline docs/common/yum.md]: Let's add a list item for **upcoming**

File: docs/common/yum.md [hunk 2]
```diff
@@ -5,122 +5,134 @@ About This Document
...
-        plugins=1
+!!! warning
+    If you have your own mirror or configuration of the EPEL repository, you **MUST** verify that the priority of the EPEL repository is either missing, or 99 or a higher number.
+    The OSG repositories must have a better (numerically lower) priority than the EPEL repositories;
+    you might have dependency resolution ("depsolving") issues otherwise.
 
-!!! note
-    If you do not have a required key you can force the installation using `--nogpgcheck=`; e.g., `yum install --nogpgcheck yum-priorities`.
 
 ### Install OSG Repositories
```
Comment (brianhlin) [inline docs/common/yum.md]: > Install the OSG repositories

File: docs/common/yum.md [hunk 3]
```diff
@@ -5,122 +5,134 @@ About This Document
...
+It is important to install and enable the YUM priorities plugin before installing grid software to avoid getting the wrong versions.
 
-### Install the Yum Repositories required by OSG
+1.  Install the YUM priorities package:
 
-The OSG RPMs currently support Red Hat Enterprise Linux 6, 7, and variants.
+        :::console
+        root@host # yum install yum-plugin-priorities
+
+2.  Ensure that `/etc/yum.conf` has the following line in the `[main]` section:
```
Comment (brianhlin) [inline docs/common/yum.md]: > 1.

File: docs/worker-node/install-wn.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ To get assistance please use this [Help Procedure](/common/help).
 Reference
 ---------
 
-Please see the documentation on using [yum and RPM](/release/yum-basics.md), [the best practices](/common/install-best-practices) for using yum to install software, and using [yum repositories](/common/yum.md).
+Please see the documentation on using [YUM and RPM](../release/yum-basics.md), and using the [OSG YUM repositories](../common/yum.md).
```
Comment (brianhlin) [inline docs/worker-node/install-wn.md]: Make these absolute links

**Pull request merged**